src/components/humanVsHumanGameComponent/humanVsHumanGameComponent.js

```diff
@@ -30,6 +30,20 @@ toggleMarker = async (event) => {
   this.setState({
     gameStatus: await PhoenixApi.getStatus(marks, next_player, current_player)
   });
+
+  if (this.state.gameStatus !== Statuses.IN_PROGRESS) {
+    this.highlightWinners(marks);
+  }
+}
+
+highlightWinners = async (marks) => {
+  let data = await PhoenixApi.getWinningIndices(marks);
```

Comment:
hmmm I don't like that `highlightWinners` both FETCHES the winning indices and it highlights them. Remember the single responsibility principle!

Reply:
^ Agree